egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -346,8 +359,9 @@ def GenerateReverberatedWavScp(wav_scp,  # a dictionary whose values are the Kal
                                                          speech_dur,  # duration of the recording
                                                          max_noises_recording  # Maximum number of point-source noises that can be added
                                                          )       
-            
-            if reverberate_opts == "":
+
+            # prefix with index 0, e.g. rvb0_swb0035, stangs for the original data
```

Comment:
`# prefix using index 0 is reserved for original data e.g. rvb0_swb0035 corresponds to the swb0035 recording in original data`